bounties_api/user/views.py

```diff
@@ -117,6 +123,16 @@ def get(self, request, public_address):
         fulfiller_fulfillment_acceptance = None if not user_fulfillments.count() else (
             user_fulfillments.filter(accepted=True).count() / user_fulfillments.count())
 
+        total_fulfillments_on_bounties = user_bounties.annotate(
+            fulfillments_count=Count('fulfillments')).aggregate(
+            Sum('fulfillments_count')
+        ).get('fulfillments_count__sum', None)
+
+        if total_fulfillments_on_bounties:
+            total_fulfillments_on_bounties = total_fulfillments_on_bounties
```

Comment:
I am not quite following this if check - does something special happen when assigning `total_fulfillments_on_bounties` to `total_fulfillments_on_bounties` here?

Reply:
This is good feedback, I need to remove that. Thanks @codeluggage 😄

Reply:
Also confused here :)